cosmic-settings/src/pages/bluetooth/backend/mod.rs

```diff
@@ -0,0 +1,5 @@
+use super::{Active, Adapter, Device};
+
+mod dbus;
```

Comment:
I think this whole module can go into the dbus-settings crate

Reply:
I moved as much as possible, but still kept a significant part, directly linked to local structs (`Adapter`, `Device`). Let me know if it is fine, happy to do more if you have specific suggestions.